packages/eslint-config/index.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
     },
     {
       // Pages and layouts are required to have a single root element if transitions are enabled.
-      files: ['**/pages/**/*.{js,ts,vue}', '**/layouts/**/*.{js,ts,vue}'],
+      files: ['**/pages/**/*.{js,ts,vue}', '**/layouts/**/*.{js,ts,vue}', '**/components/*/**/*.{js,ts,vue}'],
```

Comment:
This section is `vue/no-multiple-template-root` rule . Isn't it on line 69?

Reply:
fixed it, you are right